src/testsuite/vec_f64_dummy.c

```diff
@@ -0,0 +1,264 @@
+/*
+ Copyright (c) [2017-18] IBM Corporation.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+
+ vec_int64_dummy.c
+
+ Contributors:
+      IBM Corporation, Steven Munroe
+      Created on: Jul 31, 2017
+ */
+
+#include <stdint.h>
+#include <stdio.h>
+#include <fenv.h>
+#include <float.h>
+#include <math.h>
+
+//#define __DEBUG_PRINT__
+
+#include "vec_f64_ppc.h"
+
+int
+test512_all_f64_nan (vf64_t val0, vf64_t val1, vf64_t val2, vf64_t val3)
+{
+  const vb64_t alltrue = { -1, -1 };
+  vb64_t nan0, nan1, nan2, nan3;
+
+  nan0 = vec_isnanf64 (val0);
+  nan1 = vec_isnanf64 (val1);
+  nan2 = vec_isnanf64 (val2);
+  nan3 = vec_isnanf64 (val3);
+/* Only newest compilers support vec_and for vector bool long long.  */
```

Comment:
What's the value of the comment?  Should there be `#ifdef`s to exclude compilation by older compilers?  This comment will obviously not be true forever.  Should a more specific compiler version be mentioned instead?